package com.revature.exception;

public class ReimbursementAlreadyResolvedException extends Exception {

	public ReimbursementAlreadyResolvedException() {
		super();
		// TODO Auto-generated constructor stub
	}

	public ReimbursementAlreadyResolvedException(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
		// TODO Auto-generated constructor stub
	}

	public ReimbursementAlreadyResolvedException(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public ReimbursementAlreadyResolvedException(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public ReimbursementAlreadyResolvedException(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}

}
